gov_uk_dashboards/components/plotly/tooltip.py

```diff
@@ -0,0 +1,21 @@
+"""tooltip"""
+from dash import html
+import dash_bootstrap_components as dbc
+
+
+def tooltip(tooltip_text, tooltip_id):
+    """Creates a tooltip for providing further information"""
+
+    return html.Span(
+        [
+            " ⓘ",
+            dbc.Tooltip(
+                tooltip_text,
+                target=tooltip_id,
+                class_name="tooltip tooltiptext",
+                style={"color": "white", "font-size": 12, "width": "10px"},
```

Comment:
What does specifying the `width` do?  I'd imagine with a tooltip it would be calculated automatically based on for example the position of the tooltip, and the amount of text inside.

Reply:
Agree,  hard coding this to 10px will not make it very dynamic. Sorry this popped up on notification. It would be better to  pass in width as a parameter in the function. Without it you should let the browser decide.

Reply:
@adrianclay  in face it maybe best to pass in alot of these settings. Espically color

Reply:
Having tested the width parameter, the width of the tooltip does not change when specifying different "_px", no visible changes occurs. So the width parameter has been removed.